cli/src/commands/commit.rs

```diff
@@ -50,6 +51,11 @@ pub(crate) struct CommitArgs {
     /// $ JJ_USER='Foo Bar' JJ_EMAIL=foo@bar.com jj commit --reset-author
     #[arg(long)]
     reset_author: bool,
+    /// Set author to the provided string
+    ///
+    /// This changes author name and email while retaining timestamp.
+    #[arg(long, conflicts_with = "reset_author")]
```

Comment:
In the issues author said:
> If more than a single --author option is specified, the operation should fail, instructing the user to specify, at most, one.

I'm not sure if this is doable with Clap.

Reply:
Since the author field is not a `Vec`, you cannot pass it multiple times so it is fine as is.

Reply:
I thought Clap would override an argument with the last instance by default, but maybe I have confused it with another crate.